src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -44,7 +45,13 @@
  *
  */
 public abstract class CommandLineProgram {
-    protected final Logger logger = LogManager.getLogger(this.getClass());
+    protected transient Logger logger = LogManager.getLogger(this.getClass());
+
+    private void readObject(java.io.ObjectInputStream in)
+            throws IOException, ClassNotFoundException {
+        in.defaultReadObject();
+        logger = LogManager.getLogger(this.getClass()); // Logger is not serializable (even by Kryo)
```

Comment:
@cmnbroad I'm not sure about the serialization cost (once per task, so maybe not a problem). The issue here is that `FeatureManager` has a `CommandLineProgram` field, and we want to broadcast `FeatureManager`. In the original PR I made the field transient, which seemed to work OK, but changed it after review feedback. We could revisit this if you think it's worth looking at.